### ISSUE TITLE
Can't find .head

### ISSUE THREAD

**Issue by omahane** [stubbornella/oocss#70]
I could be nuts or blind or something but I can't seem to find any head class in the css of this files. I even looked at Walt's example that Ms. Sullivan used for the code review, but I don't see any .head in any css there either.

Could someone please tell me what I'm not seeing that they do?

**Comment by tomtomsen** [comment]
You are right, there is non. I guess that's because .head doesn't need to be styled. If you need some for your project template/template.css is the place to be  :)
